R/tests/testthat/test_generate_strings.R

```diff
@@ -0,0 +1,175 @@
+# R. Samman
+# November 07 2024
+
+rm(list = ls())
+
+if (!require(testthat)) install.packages('testthat')
+library(testthat)
+
+source("/Users/raneemsamman/Documents/GitHub/paRsynth/R/generate_strings.R")
+
+# Checking that the number of groups and individuals is correct
+
+    # what do we want to do?
+        # 1. check n_groups
+        # 2. check n_individuals (per group?)
+        # 3. n_calls (i am adding this here bc it will be a double test for n_individuals)
+    # how are we doing that?
```

Comment:
Can you add these as comments above the respective lines of code, and remove these notes?